doc/user_guide/find_package.rst

```diff
@@ -14,13 +14,14 @@ simultaneously, use
                cpp_common rostime roscpp_traits
                roscpp_serialization sensor_msgs)
   include_directories(${catkin_INCLUDE_DIRS})
+  add_definitions(${catkin_DEFINITIONS})
 
   add_executable(myexec ...)
-  target_link_libraries(${catkin_LIBRARIES})
+  target_link_libraries(myexec ${catkin_LIBRARIES})
```

Comment:
Please create a separate PR for this fix (and the one below) since they can be directly merged.